style.css

```diff
@@ -2,7 +2,7 @@
 Theme Name: MIT Libraries Child NEW
 Author: Lightning Trumpet & MIT Libraries
 Author URI: http://wordpress.org/
-Version: 2.2.1-@@branch-@@commit
+Version: 2.2.2-@@branch-@@commit
```

Comment:
Oh good, I was hoping that we bumped the theme version with this PR.

I'm not sure we have guidelines for when each number gets incremented - for this, I'm tempted to say we should make it 2.3.0, but there's time to have those conversations while we consider site redevelopment.